src/transaction/log_replication.cpp

```diff
@@ -26,20 +26,25 @@
 #include "thread_looper.hpp"
 #include "thread_manager.hpp"
 #include "transaction_global.hpp"
+#include "util_func.h"
 
 #include <cassert>
 #include <chrono>
 #include <functional>
 
 namespace cublog
 {
-
   replicator::replicator (const log_lsa &start_redo_lsa)
     : m_redo_lsa { start_redo_lsa }
   {
     log_zip_realloc_if_needed (m_undo_unzip, LOGAREA_SIZE);
     log_zip_realloc_if_needed (m_redo_unzip, LOGAREA_SIZE);
 
+    // before initializing any threads, init stats
+    cubthread::entry *const thread_entry = &cubthread::get_entry ();
+    // initialization is always performed
+    perfmon_set_stat (thread_entry, PSTAT_SC_REPL_DELAY, 0, false);
+

```

Comment:
All stats are initialized as 0, not required.
```suggestion
```

Reply:
Done